Makefile

```diff
@@ -32,18 +33,9 @@ GITHUB_ACCESS_TOKEN := $(shell cat /builder/home/git-token 2> /dev/null)
 PEGOMOCK_PACKAGE := github.com/petergtz/pegomock/
 CGO_ENABLED = 0
 
-all: build
-full: check
-check: build test
+# set dev version unless VERSION is explicitly set via environment
+VERSION ?= $(shell echo "$$(git describe --abbrev=0 --tags 2>/dev/null)-dev+$(REV)" | sed 's/^v//')
```

Comment:
@jstrachan, only set VERSION if not explicitly specified via the environment. For the case VERSION is not set use `git describe`.